packages/react-components/src/organisms/TeamCreateOutputIdentifier.tsx

```diff
@@ -0,0 +1,139 @@
+import { useCallback, useEffect, useMemo } from 'react';
+import {
+  ResearchOutputType,
+  ResearchOutputIdentifierType,
+  researchOutputToIdentifierType,
+} from '@asap-hub/model';
+import { LabeledDropdown, LabeledTextField } from '../molecules';
+import { noop } from '../utils';
+
+const getIdentifiers = (
+  researchOutputType: ResearchOutputType,
+  required: boolean,
+): Array<{
+  value: ResearchOutputIdentifierType;
+  label: ResearchOutputIdentifierType;
+}> => {
+  let identifiers = researchOutputToIdentifierType[researchOutputType] ?? [];
+
+  if (required) {
+    identifiers = identifiers.filter(
+      (identifier) => identifier !== ResearchOutputIdentifierType.None,
+    );
+  }
+
+  return identifiers.map((identifier) => ({
+    value: identifier,
+    label: identifier,
+  }));
+};
+
+const identifierMap = {
+  [ResearchOutputIdentifierType.DOI]: {
+    helpText: 'Your DOI must start with 1 and it cannot be a URL',
+    placeholder: 'DOI number e.g. 10.5555/YFRU1371',
+    regex: '^(doi:)?\\d{2}\\.\\d{4}.*$',
```

Comment:
I think these regex's should be in the validation package so the BE can potentially reuse them

Reply:
Why are they a string instead of a regular regex?

Reply:
@kmaid because the ajv and html requires a string instead of a regex.
Unfortunately, I won't have time to adapt this anymore.